qa/tasks/deepsea.py

```diff
@@ -151,7 +169,9 @@ def setup(self):
             '--no-recommends',
             run.Raw('$(rpmspec --requires -q -v DeepSea/deepsea.spec | grep manual | awk \'{print $2}\')')
             ])
+        self.setup_salt()
 
+    def setup_salt(self):
```

Comment:
Name is not fitting what it does. This function only lists minions that talk to the master.
maybe ```log_salt_setup()```?

Reply:
I don't mind.